common/test/acceptance/pages/lms/teams.py

```diff
@@ -248,7 +250,10 @@ def click_create_team_link(self):
         query = self.q(css=CREATE_TEAM_LINK_CSS)
         if query.present:
             query.first.click()
-            self.wait_for_ajax()
+
+        # This will bring you to the team management page
+        tm_page = TeamManagementPage(self.browser, self.course_id, self.topic)
```

Comment:
Nit: don't use abbreviations. I'd call this ``team_management_page`` or ``management_page`` or even just ``page``.